### PR TITLE
Have Ack just extend fmt.Stringer

### PR DESCRIPTION
**Pull request by peter-edge**
Heh

### CODE REVIEW

File: ack.go [hunk 1]
```diff
@@ -20,7 +20,9 @@
 
 package yarpc
 
-// Ack represents and acknowledgement from a oneway request
+import "fmt"
+
+// Ack represents and acknowledgement from a oneway request.
```
Comment (abhinav) [inline ack.go]: > an acknowledgement

File: api/transport/ack.go [hunk 2]
```diff
@@ -20,7 +20,9 @@
 
 package transport
 
-// Ack represents and acknowledgement from a oneway request
+import "fmt"
+
+// Ack represents and acknowledgement from a oneway request.
```
Comment (abhinav) [inline api/transport/ack.go]: an

**Comment by abhinav** [comment]
Thanks

File: ack.go (hunk 1 above)
Comment (peter-edge) [inline ack.go]: Done

File: api/transport/ack.go (hunk 2 above)
Comment (peter-edge) [inline api/transport/ack.go]: Done

**Comment by peter-edge** [comment]
Comments fixed and travis fixed

**Pull request merged**